src/backend/core/models.py

```diff
@@ -712,10 +759,44 @@ def computed_link_role(self):
         """Actual link role on the document."""
         return self.computed_link_definition["link_role"]
 
+    def _compute_abilities_cache_key(self):
```

Comment:
nit: "compute" what a big word for a simple function... "get"? or even simply a property?